js/src/personal/haoking/Haoking.module.css

```diff
@@ -9,8 +41,38 @@
 }
 
 .spotify {
- border-radius:12px;
+  border-radius: 12px;
   width: 100%;
   height: 152px;
-    border:none;
+  border: none;
+}
+
+.link {
+  text-decoration: none;
+  color: #d1d1d1;
+  font-size: 20px;
+  font-family: Helvetica, sans-serif;
 }
+
+.link_current {
+  text-decoration: none;
+  color: #fff;
+  font-size: 20px;
+  font-family: Helvetica, sans-serif;
+}
+
+a.link:hover {
```

Comment:
I wonder if this applies to all pages or just yours.
It might be good to put this under a css styled class so then it won't leak out.

Reply:
I tried testing it and it didn't seem to have any affect on other people's pages. I don't think it would because they wouldn't be importing my own css file. 

Also, can you clarify what you mean by css styled class?

Reply:
So like inside of <MantineProvider>, wrap the rest of your pages with <div with classname="styles.body"

Changs this to `.body a.link:hover {` so that it's definitely gated to your own pages.

Reply:
Added .hkbody selector to all my css classes and wrapped all pages with a div with class hkbody on new commit